include/courtier/GWebsocketConsumerT.hpp

```diff
@@ -1508,6 +1513,445 @@ class GWebsocketConsumerT
 	 //-------------------------------------------------------------------------
 };
 
+
+// Add just another consumer
+
+template<typename processable_type>
+class GWebsocketConsumerPT
+	: public Gem::Courtier::GBaseConsumerT<processable_type> // note: GBaseConsumerT<> is non-copyable
+	  , public std::enable_shared_from_this<GWebsocketConsumerPT<processable_type>>
+{
+	 //-------------------------------------------------------------------------
+	 // Simplify usage of namespaces
+	 using error_code = boost::system::error_code;
+
+public:
+	 //-------------------------------------------------------------------------
+	 /** @brief The default constructor */
+	 GWebsocketConsumerPT() = default;
+  
+         GWebsocketConsumerPT(int io_context_pool_size)
+	   : m_io_contexts(io_context_pool_size)
+	   , m_acceptor(m_io_contexts.get())
+	   , m_socket(m_io_contexts.get()){
+	   std::cout << "-I- GWebsocketConsumerPT() created with pool size: " <<  io_context_pool_size  << std::endl;
+	 }
+	 //-------------------------------------------------------------------------
+	 // Deleted copy-/move-constructors and assignment operators.
+	 GWebsocketConsumerPT(const GWebsocketConsumerPT<processable_type>&) = delete;
```

Comment:
This for me is the proper way to make a class non-copyable. I had seen in another file (GIoContexts?) the use of boost::noncopyable. Could have still been "inherited" from my code (then it is my fault :-) ) Nevertheless please eliminate boost::noncopyable when you see it and relplace with =delete constructs.